playbooks/roles/csi-driver-config/defaults/main/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Any reason for creating a dir `main`? It could be `playbooks/roles/csi-driver-config/defaults/main.yaml` OR `playbooks/roles/csi-driver-config/defaults/main/install.yaml`